shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -469,9 +467,14 @@ - (void)setBackgroundColor:(NSColor*)color {
   [_flutterView setBackgroundColor:_backgroundColor];
 }
 
-- (FlutterViewId)viewId {
+- (FlutterViewIdentifier)viewIdentifier {
+  NSAssert([self attached], @"This view controller is not attached.");
+  return _viewIdentifier;
+}
+
+- (BOOL)isImplicitView {
   NSAssert([self attached], @"This view controller is not attached.");
-  return _viewId;
+  return self.viewIdentifier == kFlutterImplicitViewId;
```

Comment:
Should kFlutterImplicitViewId be renamed to kFlutterImplicitViewIdentifier?

Reply:
It's an internal variable so we can keep it more brief, and I kind of want to keep the variable name aligned with the counterparts in other platforms. Should I change it to Identifier too?

Reply:
Actually, there's a `kFlutterImplicitViewId` defined in `common/constants.h` and I've switched the macOS embedder to use that one. The definition in `FlutterView.h` has been removed.